src/commands/poll.js

```diff
@@ -164,19 +167,41 @@ exports.run = (client, message, args, callback) => {
                     reactions[index] = [];
                 });
 
-                exports.delayedPolls.push({
+                let delayedPollData = {
                     pollId: msg.id,
-                    createdAt: new Date(),
-                    finishesAt: finishTime,
+                    createdAt: new Date().valueOf(),
+                    finishesAt: finishTime.valueOf(),
```

Comment:
valueOf instead of date because the AdditionalMessageData Model is absolutely dumb